frontend/PropState.js

```diff
@@ -251,4 +270,9 @@ const noPropsStateStyle = (theme: Theme) => ({
   padding: '0.5rem',
 });
 
+const hooksTreeLoadingStyle = {
```

Comment:
any reason this isn't in a stylesheet? (low pri, ignore)

Reply:
Just following the (super gross) convention within the file. Old DevTools are pretty gross.